### PR TITLE
OPENEUROPA-557: Create OpenEuropa Multilingual module

### PR DESCRIPTION
**Pull request by ademarco**
### Description

The business goal of this ticket is to create a new OpenEuropa module that will deliver the following multilingual functionalities once it is enabled on a site:

- Allow site builders to make content translatable in all 24 official EU languages
- Provide a language switcher block on the header region of the OE Theme
- Make sure that URLs are translatable
- Make sure that non-latin characters in URLs are transliterated
- Make sure that administrative interface always be set to English
- Provide path prefix negotiation method
- Make sure that Portuguese code is PT

### Changelog

- Added:
- Changed:
- Deprecated:
- Removed:
- Fixed:
- Security:

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,65 @@
+{
+    "name": "openeuropa/oe_multilingual",
+    "description": "OpenEuropa Multilingual module.",
+    "type": "drupal-module",
+    "license": "EUPL-1.1",
```
Comment (voidtek) [inline composer.json]: "license": "EUPL-1.2" ?

File: tests/features/showcase.feature [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  Scenario: Visitor can navigate translatable content
+
+    Given the following "Demo translatable page" content item:
+      | Title | Page title |
+      | Body  | Page body  |
+    And the following "it" translation for the "Demo translatable page" with title "Page title":
+      | Title | Titolo pagina |
+      | Body  | Testo pagina  |
+
+    And I visit the "Page title" content
```
Comment (voidtek) [inline tests/features/showcase.feature]: When I visit the "Page title" content

File: LICENCE.txt [hunk 3]
```diff
@@ -0,0 +1,190 @@
+EUROPEAN UNION PUBLIC LICENCE v. 1.2
```
Comment (aritomelo) [inline LICENCE.txt]: I suggest looking at OPENEUROPA-547 regarding this topic.

File: composer.json [hunk 4]
```diff
@@ -0,0 +1,80 @@
+{
+    "name": "openeuropa/oe_multilingual",
+    "description": "OpenEuropa Multilingual module.",
+    "type": "drupal-module",
+    "license": "EUPL-1.2",
+    "minimum-stability": "dev",
+    "prefer-stable": true,
+    "require": {
+        "drupal/administration_language_negotiation": "1.4",
```
Comment (drupol) [inline composer.json]: I just release version 1.5 that contains good stuff, I'd suggest to update to 1.5 directly.

File: oe_multilingual.info.yml [hunk 5]
```diff
@@ -0,0 +1,42 @@
+name: Drupal module template
```
Comment (drupol) [inline oe_multilingual.info.yml]: Should we update this to something more appropriate ?
Same goes for the description hereunder ?

File: composer.json (hunk 4 above)
Comment (ademarco) [reply composer.json]: We found an issue with @drupol on 1.5, we agreed are gonna stay on 1.4 until that issue is solved, then we upgrade to the next release directly, e.g. 1.6.

File: LICENCE.txt (hunk 3 above)
Comment (ademarco) [reply LICENCE.txt]: Thanks @aritomelo we will deal with it on all projects once OPENEUROPA-547 will be in the next sprint.

File: composer.json (hunk 4 above)
Comment (drupol) [reply composer.json]: I've addressed the issue: https://www.drupal.org/project/administration_language_negotiation/issues/2897784#comment-12600524

File: LICENCE.txt (hunk 3 above)
Comment (aritomelo) [reply LICENCE.txt]: Okidocki!

**Comment by ademarco** [comment]
@imanoleguskiza current D7 language negotiation strategy will indeed change in D8 and, just for the POC, we will use the default path prefix. This will likely change in future.

**Comment by imanoleguskiza** [comment]
@ademarco if that is the case then I have nothing more to add. Good job 👍

**Pull request merged**